Sources/GRPCCore/Status.swift

```diff
@@ -0,0 +1,262 @@
+/*
+ * Copyright 2023, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/// A status object represents the outcome of an RPC.
+///
+/// Each ``Status`` is composed of a ``Status/code-swift.property`` and ``Status/message``. Each
```

Comment:
The first docc link is very interesting :D

Reply:
``Status``? 🤨

Reply:
I mean this ```Status/code-swift.property```

Reply:
Off-by-one error ;) 

Yeah, docc symbols aren't case sensitive so `Code`-the-type and `code`-the-property are ambiguous otherwise.